### DISCUSSION TITLE
Missing Customize filter fields dropdown

### DISCUSSION THREAD

**Discussion by MyNameisAndrewMcgregor** [microsoft-search/pnp-modern-search#2664] [Q&A]
I'm having an issue with Search Filters.  When I go to Customize filters and Filter field. I don't get a dropdown. I have the same issue in google chrome and edge. 

I followed the documentation and this tutorial, but It’ll no luck.
https://www.youtube.com/watch?v=ITnHsCyyBJs

![2022-12-27_12-06-49](https://user-images.githubusercontent.com/121514126/209709049-eec3d7df-1a5b-4a83-9cad-b38a9fe11a05.png)

**Comment by gretchunkim** [comment]
If your indexed properties schema is selected to be non-refinable (no filter), then they shouldn't show up here. Depends on your data source/result, check the column/field property (SharePoint), make it refinable if it's possible. Then re-index/crawl it before using the filter feature.

**Comment by wobba** [comment]
If you don't get a drop-down, just type the name manually instead.
